R/CreatePackageVignette.R

```diff
@@ -4,12 +4,9 @@
 #'    vignette can be rendered into a standard HTML vignette with the
 #'    \code{\link[knitr:vignette_engines]{knitr::rmarkdown}} vignette engine
 #'    into HTML vignettes upon package building. It is also compatible with
-#'    \code{\link[pkgdown:build_articles]{pkgdown}} sites. See the vignette
-#'    \href{https://cran.r-project.org/web/packages/pkgnet/vignettes/publishing-reports.html}{
-#'    "Publishing Your pkgnet Package Report"} for details about how to use this
-#'    function, as well as
-#'    \href{https://cran.r-project.org/web/packages/pkgnet/vignettes/pkgnet-report.html}{
-#'    our example for pkgnet}.
+#'    \code{\link[pkgdown:build_articles]{pkgdown}} sites. See examples of these vignette formatted reports as well as standard formatted reports 
+#'    at \href{https://uptakeopensource.github.io/pkgnet-gallery/docs/articles/pkgnet-gallery.html}{the pkgnet gallery}.  Find examples of how to use
+#'    a pkgnet report in the \href{https://uptakeopensource.github.io/pkgnet/articles/index.html}{pkgnet vignettes}.
```

Comment:
I don't think you should change the text here. Keep the original text, change the first link to 

- https://uptakeopensource.github.io/pkgnet/articles/publishing-reports.html

and the second link to

- https://uptakeopensource.github.io/pkgnet-gallery/exhibits/pkgnet-vignette/pkgnet-vignette.html